### PR TITLE
Feature sendinblue api setup 20

### PR DESCRIPTION
**Pull request by hapalacios**
### What this PR does (required):
* PR functionality: sendinblue API setup completed. Now, every time a new user sign up will receive a welcome email from the API.

### Screenshots / Videos (required):
- Sendinblue setup completed:

Screen Shot 2021-08-14 at 1 20 28 PM
Screen Shot 2021-08-14 at 1 20 36 PM
Screen Shot 2021-08-14 at 1 20 45 PM
Screen Shot 2021-08-14 at 1 21 03 PM
Screen Shot 2021-08-14 at 1 22 01 PM
Screen Shot 2021-08-14 at 1 25 45 PM

### Any information needed to test this feature (required):
- Signup only with new emails that didn't were used before.
- If you want to signup with the same email you need to delete the email from database, or use another db, and delete the email from the API dashboard.
- You will receive the welcome email usually within 1-5 minutes.

### Any issues with the current functionality (optional):
- No issues related with API found: the changes were made in sendWelcomeEmail.js and controllers/auth.js.
- DashboardbarMenu.tsx and useBoardMenu.tsx were crashing all the time and were commented for testing purposes.
- Login and Signup style was added again. For some reason still doesn't shown on main, probably due current PRs that are under review and presented merging conflicts.

https://user-images.githubusercontent.com/60803185/129457663-e9562c0f-7b45-4c65-ab88-93ec29b3c007.mp4

### CODE REVIEW

**Comment by hapalacios** [comment]
Recommended to install API package in server folder:
npm install sib-api-v3-sdk

File: server/controllers/utils/sendWelcomeEmail.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const SibApiV3Sdk = require('sib-api-v3-sdk');
+const defaultClient = SibApiV3Sdk.ApiClient.instance;
+
+const sendWelcomeEmail = async (to) => {
+    const email = to; 
+    console.log(email);
```
Comment (moffatethan) [inline server/controllers/utils/sendWelcomeEmail.js:6]: Could this be removed?

File: server/controllers/utils/sendWelcomeEmail.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const SibApiV3Sdk = require('sib-api-v3-sdk');
+const defaultClient = SibApiV3Sdk.ApiClient.instance;
+
+const sendWelcomeEmail = async (to) => {
+    const email = to; 
+    console.log(email); 
+    let apikey = process.env.SENDINBLUE_API_KEY
+    console.log(apikey)
```
Comment (moffatethan) [inline server/controllers/utils/sendWelcomeEmail.js:8]: This as well please

File: server/controllers/utils/sendWelcomeEmail.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    let apiInstance = new SibApiV3Sdk.ContactsApi(); 
+    let createContact = new SibApiV3Sdk.CreateContact(); 
+    createContact.email = email; 
+    createContact.listIds = [2]; 
+
+    // call SIB api 
+    apiInstance.createContact(createContact).then((data) => {
+        // success 
+        console.log('sendinblue success');
+        console.log(data);
```
Comment (moffatethan) [inline server/controllers/utils/sendWelcomeEmail.js:23]: This as well

**Review by moffatethan — APPROVED** [review]
Remove the logs but otherwise, all good

File: server/controllers/utils/sendWelcomeEmail.js (hunk 3 above)
Comment (hapalacios) [reply server/controllers/utils/sendWelcomeEmail.js]: Done.

File: server/controllers/utils/sendWelcomeEmail.js (hunk 1 above)
Comment (hapalacios) [reply server/controllers/utils/sendWelcomeEmail.js]: Done.

File: server/controllers/utils/sendWelcomeEmail.js (hunk 2 above)
Comment (hapalacios) [reply server/controllers/utils/sendWelcomeEmail.js]: Done.

**Pull request merged**